tests/Feature/api/v1/SettingsTest.php

```diff
@@ -56,10 +56,9 @@ public function testApplicationSettings()
      */
     public function testUpdateApplicationSettingsWithValidInputs()
     {
-        $imageFile = UploadedFile::fake()->image('logo.svg');
-
         $payload = [
-            'logo_file'                      => $imageFile,
+            'logo'                           => '/storage/image/asd.svg',
```

Comment:
please test with url or image file, if both are provided only the file is processed and the url is overwriten